File: HomeTask/app/src/main/java/com/example/hometask/Child.java
package com.example.hometask;

import com.j256.ormlite.field.DatabaseField;

public class Child {
    @DatabaseField(generatedId = true)
    int id;
    @DatabaseField(foreign = true, foreignAutoCreate = true)
    Client client;
    @DatabaseField(canBeNull = false)
    String nameChild;


    public Child(){
    }

    public Child(String nameChild){
        super();
        this.nameChild = nameChild;
    }

    public Child(Client client, String nameChild) {
        super();
        this.client = client;
        this.nameChild = nameChild;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNameChild() {
        return nameChild;
    }

    public void setNameChild(String nameChild) {
        this.nameChild = nameChild;
    }

    public Client getClient() {
        return client;
    }

    public void setClient(Client client) {
        this.client = client;
    }

    @Override
    public String toString() {
        return "Child{" +
                "id=" + id +
                ", client=" + client +
                ", nameChild='" + nameChild + '\'' +
                '}';
    }
}

File: HomeTask/app/src/main/java/com/example/hometask/Child.kt
package com.example.hometask

import com.j256.ormlite.field.DatabaseField

class Child {
    @DatabaseField(generatedId = true)
    var id: Int = 0
    @DatabaseField(foreign = true, foreignAutoCreate = true)
    var client: Client? = null
    @DatabaseField(canBeNull = false)
    var nameChild: String = ""


    constructor() {}

    constructor(nameChild: String) : super() {
        this.nameChild = nameChild
    }

    constructor(client: Client, nameChild: String) : super() {
        this.client = client
        this.nameChild = nameChild
    }

    override fun toString(): String {
        return "Child{" +
                "id=" + id +
                ", client=" + client +
                ", nameChild='" + nameChild + '\''.toString() +
                '}'.toString()
    }
}
